src/ApiApprover.sln

```diff
@@ -15,6 +15,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		..\build.ps1 = ..\build.ps1
 		Directory.Build.props = Directory.Build.props
 		..\GitVersion.yml = ..\GitVersion.yml
+		GitVersionWorkaround.targets = GitVersionWorkaround.targets
```

Comment:
This should be removed.

Reply:
I did remove this on master. Let me check